media-libs/nvidia-vaapi-driver/nvidia-vaapi-driver-0.0.12.ebuild

```diff
@@ -20,3 +20,13 @@ RDEPEND="media-libs/gst-plugins-bad
 DEPEND="${RDEPEND}
 	>=media-libs/nv-codec-headers-11.1.5.1"
 BDEPEND="virtual/pkgconfig"
+
+pkg_postinst() {
+	# Source: https://github.com/elFarto/nvidia-vaapi-driver/blob/master/src/backend-common.c#L13
```

Comment:
This file can change and the line would be wrong. If you want to link to the source, use the last release tag:
https://github.com/elFarto/nvidia-vaapi-driver/blob/fc5ce861fdbb9b4a67a702ba7d08e5b963584418/src/backend-common.c#L13

Reply:
Great point. Will change in next commit.